src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -156,34 +171,87 @@ public static ScheduleView CreateSchedule(Category category, string name, string
         /// <summary>
         ///     Remove Schedule Field from Schedule.
         /// </summary>
-        /// <param name="field"></param>
+        /// <param name="fields"></param>
         /// <returns name="View">View Schedule</returns>
-        public ScheduleView RemoveField(Revit.Schedules.ScheduleField field)
+        public ScheduleView RemoveField(List<Revit.Schedules.ScheduleField> fields)
         {
-            if (this.Fields.Any(x => x.Name == field.Name))
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            foreach (Revit.Schedules.ScheduleField field in fields)
             {
-                var doc = DocumentManager.Instance.CurrentDBDocument;
-                TransactionManager.Instance.EnsureInTransaction(doc);
-                this.InternalViewSchedule.Definition.RemoveField(field.InternalScheduleField.FieldId);
-                TransactionManager.Instance.TransactionTaskDone();
+                if (this.Fields.Any(x => x.Name == field.Name))
+                {
+                    this.InternalViewSchedule.Definition.RemoveField(field.InternalScheduleField.FieldId);
+                }
             }
 
+            TransactionManager.Instance.TransactionTaskDone();
+
             return this;
         }
 
         /// <summary>
         ///     Add Field.
         /// </summary>
-        /// <param name="field"></param>
+        /// <param name="fields"></param>
         /// <returns></returns>
-        public ScheduleView AddField(Revit.Schedules.SchedulableField field)
+        public ScheduleView AddFields(List<Revit.Schedules.SchedulableField> fields)
         {
-            if (!this.Fields.Any(x => x.Name == field.Name))
+            var doc = DocumentManager.Instance.CurrentDBDocument;
+            TransactionManager.Instance.EnsureInTransaction(doc);
+
+            foreach (Revit.Schedules.SchedulableField field in fields)
+            {
+                if (!this.Fields.Any(x => x.Name == field.Name))
+                {
+                    this.InternalViewSchedule.Definition.AddField(field.InternalSchedulableField);
+                }
+            }
+
+            TransactionManager.Instance.TransactionTaskDone();
+
+            return this;
+        }
+
+        /// <summary>
+        ///     Export View Schedule to CSV, TSV etc.
+        /// </summary>
+        /// <param name="path">A valid file path with file extension.</param>
+        /// <param name="exportOptions">Export Options. If null, default will be used.</param>
+        /// <returns></returns>
+        public ScheduleView Export(
+            string path, 
+            [DefaultArgumentAttribute("null")] ScheduleExportOptions exportOptions)
```

Comment:
it's not how you get the null, that I'm opposed to. It's that I don't think the default value should ever be null.